inc/color-picker.php

```diff
@@ -0,0 +1,119 @@
+<?php
```

Comment:
Can there be a file info section added in here please?
ie:
```
/**
 * Set the theme colors
 *
 * @package Nightingale-2-0
 * @copyright NHS Leadership Academy, Tony Blacker
 * @version 1.1 21st August 2019
 */
```
(with your details obviously ;) )